sdcm/cluster_gce.py

```diff
@@ -12,11 +12,11 @@ class CreateGCENodeError(Exception):
 
 
 def gce_create_metadata(extra_meta=None):
-    tags = cluster.create_common_tags()
-    tags['startup-script'] = cluster.Setup.get_startup_script()
+    meta = cluster.create_common_tags()
     if extra_meta:
-        tags.update(extra_meta)
-    return tags
+        meta.update(extra_meta)
+    tags_list = [{'key': str(k), 'value': str(v)} for k, v in meta.items()]
```

Comment:
No need to change the format, you can pass a dictionary to `ex_set_node_metadata`
https://libcloud.readthedocs.io/en/latest/compute/drivers/gce.html#libcloud.compute.drivers.gce.GCENodeDriver.ex_set_node_metadata
It wraps the metadata with the `_format_metadata` method that supports dict too.